core.ts

```diff
@@ -49,3 +49,17 @@ export function validateEvent<T>(event: T): event is T & UnsignedEvent {
 
   return true
 }
+
+/**
+ * Sort events in reverse-chronological order by the `created_at` timestamp,
+ * and then by the event `id` (lexicographically) in case of ties.
+ * This mutates the array.
+ */
+export function sortEvents(events: Event[]): Event[] {
+  return events.sort((a: NostrEvent, b: NostrEvent): number => {
+    if (a.created_at !== b.created_at) {
+      return b.created_at - a.created_at
+    }
+    return a.id.localeCompare(b.id)
```

Comment:
@vitorpamplona re: https://github.com/nostr-protocol/nips/issues/759#issuecomment-1704539371 and https://github.com/nostr-protocol/nips/issues/759#issuecomment-1705335513, does my sort function match yours? I'm sorting by `created_at` desc, then by `id` asc. 

Are there any arguments for which direction to sort `id`? I guess events with proof of work would be sorted above others.

Reply:
I do both desc but I can change.